src/services/jsTyping.ts

```diff
@@ -53,6 +53,7 @@ namespace ts.JsTyping {
      */
     export function discoverTypings(
         host: TypingResolutionHost,
+        log: ((message: string) => void) | undefined,

```

Comment:
should this just be `((message: string) => void)` with default parameter of function that do nothing.... so we don't have to check whether the log parameter is defined